scripts/test/run-integration-tests.sh

```diff
@@ -34,6 +34,10 @@ KUBE CONFIG: $KUBE_CONFIG_PATH
 CLUSTER_NAME: $CLUSTER_NAME
 REGION: $REGION"
 
+if [[ -n "${ENDPOINT}" ]]; then
```

Comment:
I think we can move this logic inside cluster.sh. Since that script is used by run-canary-tests.sh as well so would be good to have common logic in 1 place.

Reply:
Makes sense, will do that.